crates/re_ui/src/lib.rs

```diff
@@ -1315,12 +1309,41 @@ pub fn drop_down_menu(
     egui::ComboBox::from_id_source(id_source)
         .selected_text(selected_text)
         .show_ui(ui, |ui| {
-            let background_x_range = (ui.max_rect() + ui.spacing().menu_margin).x_range();
-
             list_item::list_item_scope(ui, "inner_scope", |ui| {
-                full_span::full_span_scope(ui, background_x_range, |ui| {
-                    content(ui);
-                });
+                content(ui);
             });
         });
 }
+
+/// Rerun custom extensions to [`egui::Ui`].
+// TODO(#4569): move everything here
+pub trait UiExt {
+    fn ui(&self) -> &egui::Ui;
+    fn ui_mut(&mut self) -> &mut egui::Ui;
+
+    /// Retrieve the current full-span scope.
+    fn get_full_span(&self) -> egui::Rangef {
```

Comment:
```suggestion
    fn full_span(&self) -> egui::Rangef {
```
would be the more rusty name. counter argument is that this sounds like drawing an ui element then 🤔